mono/tests/Makefile.am

```diff
@@ -741,32 +741,10 @@ endif
 
 if ARM
 PLATFORM_DISABLED_TESTS += filter-stack.exe weak-fields.exe
-INTERP_PLATFORM_DISABLED_TESTS= \
-	assemblyresolve_event6.exe \
-	delegate-async-exit.exe \
-	delegate-delegate-exit.exe \
-	delegate-exit.exe \
-	delegate1.exe \
-	delegate3.exe \
-	thunks.exe \
-	finalizer-exception.exe \
-	delegate5.exe \
-	delegate8.exe \
-	bug-323114.exe \
-	threadpool-exceptions3.exe \
-	threadpool-exceptions6.exe \
-	bug-80392.2.exe \
-	remoting3.exe \
-	remoting2.exe \
-	appdomain-unload-callback.exe \
-	monitor-abort.exe
```

Comment:
I thought a bunch of those failures are due to the "`r8` problem" in the native2managed transition. @BrzVlad this isn't fixed yet, right?

Reply:
What is the expected failure in this case? Are the tests suppose to be flaky, or are they suppose to crash?

Reply:
It depends on the C compiler, that is, if it uses `r8` in certain scenarios. For example, on my machine it only happened with `-O2`, but not with `-O0` with a specific version of gcc. Once you have a faulty build, it crashes deterministic.

As far as I remember, we changed the Linux distribution on the ARM bots since then, so it might "works" now 😉 If so, we could also enable `mcs` tests on ARM: https://github.com/mono/mono/blob/2c2fac29ebc4614ca3786d30d582ed8db906a0cc/scripts/ci/run-test-interpreter.sh#L11

Reply:
@luhenry @lewurm I think it is better to be on the safe side and keep arm tests disabled for now. IIRC at some point some unrelated interp changes started triggering some of these failures, so they are a little bit unpredictable.

Reply:
Ok, so which tests exactly should we keep disabled? All of them?

Reply:
Yes. We can reenable them all together once we fix that issue. Maybe add a comment, that these are possible to fail due to r8 clobbering, so it's known why they are there.

Reply:
I added back all the previously disabled tests to ARM.